src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfirm.tsx

```diff
@@ -92,15 +94,29 @@ export const LimitOrderConfirm = () => {
 
   const handleConfirm = useCallback(async () => {
     const quoteId = activeQuote?.response.id
-    if (!quoteId) {
+    const accountId = activeQuote?.params.accountId
+
+    if (!quoteId || !accountId) {
       return
     }
 
     // TEMP: Bypass allowance approvals and jump straight to placing the order
     setLimitOrderInitialized(quoteId)
     confirmSubmit(quoteId)
     await placeLimitOrder({ quoteId, wallet })
-  }, [activeQuote?.response.id, confirmSubmit, placeLimitOrder, setLimitOrderInitialized, wallet])
+    // refetch the orders list for this account
+    queryClient.invalidateQueries({
+      queryKey: ['getLimitOrdersForAccount', accountId],
+    })
```

Comment:
[`invalidateQueries` defaults to invalidating active only](https://tanstack.com/query/latest/docs/reference/QueryClient#queryclientinvalidatequeries), but since the `<LimitOrderConfirm />` component is a different screen altogether and not a modal on top of `<LimitOrderList />` like `<CancelLimitOrder />` then there will be no active subscribers here and this will be a no-op 

```suggestion
    queryClient.invalidateQueries({
      queryKey: ['getLimitOrdersForAccount', accountId],
      refetchType: 'all',
    })
```

Reply:
Thank you ser. Confirmed that solves the issue:
https://jam.dev/c/1a0bb973-7fbb-45b8-9278-5c94033eea0f

Reply:
LFG